joinmarket-webui/jam
# dev(setup): switch to docker compose v2

`docker-compose` (v1) is deprecated. `docker compose` (v2) should be used.
See [Migrate to Compose V2](https://docs.docker.com/compose/migrate/).

## How to test
Please verify that starting/stopping the development environment is still working for you.
- `npm run regtest:build`
- `npm run regtest:up`
- `npm run regtest:down`
- `npm run regtest:clear`

Tested with `docker v26.1.4`.
```
[user@host ~]$ docker --version
Docker version 26.1.4, build 5650f9b
```
